migrator/migrations/n_09_to_n_10_postgres_cluster_cves/legacy/store.go

```diff
@@ -0,0 +1,19 @@
+// This file was originally generated with
```

Comment:
Do we need to migrate cluster cve edge store? It looks like we can migrate cluster cve edge store only with associated cves. We may not need to migrate cluster cve store. @connorgorman @md2119 What is your suggestion? Or we do not migrate any at all?

Reply:
Let's just do it

Reply:
The cluster cves migration is already included here. It looks like we do not need to migrate cluster cve edges because they are all populated in initialization time.

Reply:
Yes, the cluster-cve edges do not have state that needs to carried over through migration.